prototype2/gdgem/vmm3/CalibrationFile.cpp

```diff
@@ -52,8 +45,8 @@ void CalibrationFile::loadCalibration(std::string jsonstring) {
   try {
     Root = nlohmann::json::parse(jsonstring);
   } catch (...) {
-    LOG(INIT, Sev::Warning, "Invalid Json file: {}", jsonstring);
-    return;
+    LOG(INIT, Sev::Warning, "Invalid Json: {}", jsonstring);
+    throw std::runtime_error("Invalid Json in calibration file.");
```

Comment:
Consider using c++11 nested exceptions, to include the underlying exception, in case whoever catches this wants to extract more info about the problem.

Reply:
Please elaborate - more info than what? Maybe a code example?